routes/accounts.js

```diff
@@ -17,14 +18,31 @@ router.get( '/register', ( request, response ) => {
   response.render( 'accounts/register' )
 })
 
-router.post( '/register', ( request, response , next) => {
+router.post( '/register', ( request, response, next ) => {
   const User = request.app.get( 'models' ).User
 
   const { email, password } = request.body
 
-  User.create({ email, password: encryptPassword( password ) })
-    .then( user => RegistrationEmail.send( user ))
-    .then( user => response.redirect( '/accounts/verify' ))
+  const registerUser = (email, password) => {
+    User.create({ email, password: encryptPassword( password ) })
+      .then( user => RegistrationEmail.send( user ))
+      .then( user => response.redirect( '/accounts/verify' ))
+  }
+
+  validateEmail(email)
+  .then(registerUser(email, password), (error, email) => {
```

Comment:
`.then` should be indented under parent promise.  Please insert spaces between parens and params, and remove braces for single line function. I'm also tempted to move the rejection function out to make this code a little clearer:
```js
const emailValidationFailed = (error, email) =>
  response.render( 'accounts/register', { error, email } )

validateEmail( email )
  .then( registerUser( email, password ), emailValidationFailed )
```